models.py

```diff
@@ -559,11 +560,26 @@ def _urls_and_domains(self, auth_entity, user_url):
     for i, url in enumerate(candidates):
       final, domain, ok = util.get_webmention_target(url, resolve=i < MAX_AUTHOR_URLS)
       if ok:
-        if util.schemeless(final.lower()).startswith(util.schemeless(url.lower())):
+        current_url = ''
+        final = final.lower()
+        # Ignore redirects when fetching the root url, they are dealt with next.
```

Comment:
hmm. we probably still want do this new rel-me check even after a lateral redirect, though, right? e.g. if we start with`/foo` and get redirected to `/bar`?

Reply:
yes, thanks for pointing it out. I realised there's another case too, going from a url with a path to a deeper path... then I realised all redirects are actually just a pre-check, so the whole thing became simpler! :-)